Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -46,25 +46,28 @@ def dispatch_build_bottle
     repo.gsub!("linux", "home") unless args.tap
 
     if (macos = args.macos)
-      # Fixup version for ARM/Apple Silicon
-      # TODO: fix label name to be 11-arm64 instead and remove this.
-      macos.gsub!(/^11-arm$/, "11-arm64")
-
-      macos = macos.yield_self do |s|
-        MacOS::Version.from_symbol(s.to_sym)
-      rescue MacOSVersionError
-        MacOS::Version.new(s)
+      # We accept runner name syntax (11-arm64) or bottle syntax (arm64_big_sur)
+      os, arch = macos.yield_self do |s|
+        tag = Utils::Bottles::Tag.from_symbol(s.to_sym)
+        [tag.to_macos_version, tag.arch]
+      rescue ArgumentError, MacOSVersionError
+        os, arch = macos.split("-", 2)
+        [MacOS::Version.new(os), arch&.to_sym]
       end
 
-      # Fixup label for ARM/Apple Silicon
-      macos_label = if macos.arch == :arm64
-        # TODO: fix label name to be 11-arm64 instead.
-        "#{macos}-arm"
+      # TODO: fix label name to be 11-arm64 instead.
```

Comment:
@Bo98 could you open an issue for that (or I can, if desired)?

Reply:
#11076